cdk/instance-startup.sh

```diff
@@ -4,11 +4,17 @@ adduser --disabled-password content-api
 
 cd /home/content-api
 
-aws --region eu-west-1 s3 cp s3://content-api-dist/content-api-sanity-tests/${Stage}/sanity-tests/sanity-tests_1.0_all.deb /home/content-api/sanity-tests_1.0_all.deb
+# shellcheck disable=SC2016
```

Comment:
This `shellcheck` directive was an interesting distraction. Why don't we want the values expanded in the `aws` line below? Are we adding single quotes because something else demands them?

Reply:
I put the quotes there because of the `${Stage}` expansion, to not confuse that with an environment variable.

Then, I realised that actually of course that value is substituted _before_ the script is executed, so technically it's not needed.

Then I thought I should leave it, because theoretically it's possible for a "nasty" to get into the expression from the `${}` substitution (while it's supplied from the CDK if you're being strict it's still kinda "untrusted" data) so I thought it was worth keeping in.

I always prefer single-quoting rather than double-quoting unless I explicitly _need_ things to get expanded

Reply:
Fair enough 😄